### DISCUSSION TITLE
表建好后如何修改分桶数？

### DISCUSSION THREAD

**Discussion by gaoyan1998** [apache/doris#25048] [A - General / Q&A]
建表时候没想好，设置少了，现在想增加分桶数量怎么办，单分区，官网文档并没有找到方法，版本2.0

**Comment by ixzc** [comment]
You can refer to this:
https://doris.apache.org/zh-CN/docs/dev/advanced/partition/table-temp-partition#%E6%9C%80%E4%BD%B3%E5%AE%9E%E8%B7%B5
